x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/action/EsqlResponseListener.java

```diff
@@ -27,14 +30,21 @@ public class EsqlResponseListener extends RestResponseListener<EsqlQueryResponse
     private final RestChannel channel;
     private final RestRequest restRequest;
     private final MediaType mediaType;
+    /**
+     * Keep the initial query for logging purposes.
+      */
```

Comment:
One extra space here. Does the formatter hate this?

Reply:
Weird, doesn't seem to bother the formatter. I'll fix it manually.